README.md

```diff
@@ -50,7 +66,9 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
 ```
 
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
+
+The UserApp namespace isn't declared anywhere.
```

Comment:
I can see why you'd say that's a problem, but it's not the biggest problem. As we said in class, the biggest problem is that `UserApp.add_all_users(data);` needs to be inside the `.done()` callback in order to have access to `data`.